### DISCUSSION TITLE
"GitHub Copilot requires VS Code 1.57+. Please update your VS Code" Using Code OSS 1.62.0

### DISCUSSION THREAD

**Discussion by raahimfareed** [community/community#10292] [Copilot Conversations]
Using Code OSS on Arch Linux.
Code OSS Version 1.62.0
Github Copilot Version v1.7.4688

I have added `"enable-proposed-api":["github.copilot"]` in the json file after running `Preferences: Configure Runtime Arguments` but no success

**Comment by davecheney** [comment]
I’m sorry, you need to use VSCode, not one of its derivatives.
